qiskit_nature/second_q/algorithms/excited_states_solvers/qeom.py

```diff
@@ -509,28 +546,105 @@ def _build_eom_matrices(
         logger.debug("\nM:=========================\n%s", m_mat)
         logger.debug("\nV:=========================\n%s", v_mat)
 
-        return m_mat, v_mat, q_mat, w_mat, m_mat_std, v_mat_std, q_mat_std, w_mat_std
+        # Matrix building rules
+        # h_mat = [[M, Q], [P, N]] and s_mat = [[V, W], [T, U]]
+        # N = M.conj() = M.T
+        # P = Q.adjoint() = Q.conj()
```

Comment:
```suggestion
        # P = Q.adjoint() = Q.T.conj()
```

Reply:
As stated above, Q = Q.T. Maybee I should still remind it

Reply:
Okay I see. The reason I made the change suggestion here is because this is what you implemented in the code below :+1: